src/products/review/signoffs.md

```diff
@@ -19,16 +21,22 @@ Sign-offs on Beta are not possible through the entire cycle: sign-offs deadline
 
 While it’s still possible to take last minute sign-offs during the rest of the third week, for example to fix bugs or improve coverage for tier 1 languages. that comes with increased risk, since there will be no time for another beta build to test these changes. Once the code merges from Beta to Release, any sign-off update would require a manual uplift to `mozilla-release` and a new Release Candidate (RC) build.
 
-Beta sign-offs are performed on l10n.mozilla.org (Elmo) [in this page](https://l10n.mozilla.org/shipping/dashboard?tree=fennec_beta&tree=fx_beta).
-
 Given that the Beta version is closed to sign-offs for more than 10 days, to avoid accumulating a lot of review backlog, the PM in charge can decide to take sign-offs on the [Nightly version](https://l10n.mozilla.org/shipping/dashboard?tree=fennec_central&tree=fx_central) for the remaining part of the cycle.
 
-![Beta cycle](../../assets/images/signoffs/beta_and_nightly.png)
+![Timeline of all channels, 2 cycles](../../assets/images/signoffs/all_channels_timeline.png)
 
 As explained at the beginning, sign-offs are not used for Nightly builds, but this approach has a few benefits:
-* It reduces the content to review at the beginning of the Beta cycle. That's particularly important when it comes to the amount of changes generated for Firefox desktop by [Fluent migrations](../firefox_desktop/fluent_migrations.md).
+* It reduces the content to review at the beginning of the Beta cycle. That’s particularly important when it comes to the amount of changes generated for Firefox desktop by [Fluent migrations](../firefox_desktop/fluent_migrations.md).
 * Since sign-offs are tied to a version number, when Nightly moves to Beta, the new Beta version will already have updated sign-offs and won’t fall back to outdated changesets. As a consequence, the first beta of the cycle will ships with better l10n coverage. In the example above, you start taking sign-offs for 77 at the end of the Nightly cycle. When 77 moves to Beta, it will already have update content compared to the last 76 sign-offs.
 
+### Updating release
+
+When the code moves from `mozilla-beta` to `mozilla-release`, `l10n-changesets.json` is frozen, as `l10n-bumper` is not configured to run against the release branch, and Elmo doesn’t allow to sign off changesets on the release channel.
+
+In case of severe issues affecting one or more locales, it’s still possible to manually update the shipping changesets. A patch needs to be provided for `l10n-changesets.json` in `mozilla-release` branch and approved for uplift by Release Drivers. Note that a dot release is needed in order to ship the updated version to users.
```

Comment:
This seems vague, but I tried to find examples, but I couldn't find any. All the cases I've found were still using Ship-It, and it was a matter of asking people to update the shipping changeset.

Reply:
And I've figured out how to find one right after writing this comment 🤦